packages/tools/update-readme-contributors.ts

```
@@ -12,22 +12,27 @@ const readmePath = `${rootDir}/README.md`;
 const { insertContentIntoFile } = require('./tool-utils.js');
 
 async function gitHubContributors(page: number): Promise<Contributor[]> {
```

ESLint

[error] 'page' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)